BookWorm/BookWorm/View Controllers/GroupChatViewController.m

```diff
@@ -32,7 +32,7 @@ - (void)viewDidLoad {
     [self fetchFromParse];
     
     // using live query to immediately show the change
-    self.liveQueryClient = [[PFLiveQueryClient alloc] initWithServer:@"wss://bookworm.b4a.io" applicationId:@"cfEqijsSr9AS03FR76DJYM374KHH5GddQSQvIU7H" clientKey:@"F9dLUvMhb8D7aMCAukUDMFae630qhhlYTki6dGxP"];
+    self.liveQueryClient = [[PFLiveQueryClient alloc] initWithServer:@"wss://bookwormnew.b4a.io" applicationId:@"uJmLEb8geUHYaMmMWTk3rBlYQJ0L2MzIfaCPwFrh" clientKey:@"4oKQmiYRyr49sVDqLo9TJDUIU9FYRwUfCgRKBKjW"];
```

Comment:
nit: given server url, applicationId and clientKey are used in several places, we could put them into const values in one place, and use the const value variable at callsites, to minimize any risks to change break while upgrading Parse server connections